lib/Runtime/Library/DateImplementation.h

```diff
@@ -514,20 +507,21 @@ namespace Js {
             // check the IsDaylightSavings?
             if (ptzd->fDst == false)
             {
-                const char16 *const standardName = scriptContext->GetTimeZoneInfo()->StandardName;
-                bs->AppendChars(standardName, static_cast<CharCount>(wcslen(standardName)));
+                size_t nameLength;
+                const WCHAR *const standardName = DateTime::DaylightTimeHelper::GetStandardName(&nameLength);
```

Comment:
I dont think you can change this API to use these semantics- the timezone information is cached per script context today (actually an MRU cache with a max count of 2). Now you have this changed to a static. This has perf implications but more importantly, script can be executed on multiple threads today and using a unprotected static can lead to race conditions/incorrect results. Same with GetDaylightTime. I recommend just getting the data from the script context since you still have a cache there.